exercises/practice/resistor-color-trio/.meta/example.gd

```diff
@@ -0,0 +1,18 @@
+func color_code(colors):
+    var colors_code = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
```

Comment:
Could you please change space indentation to tab indentation? I promise that at some point I will write a small CI script to detect spaces, but for now we need to check this manually :)

GDScript will work with both spaces and tabs, as long as they are used consistently. However, the main programming environment of GDScript is Godot Editor, and it uses tabs by default.